testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -103,6 +105,15 @@ pub struct MinedNakamotoBlockEvent {
     pub tx_events: Vec<TransactionEvent>,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct PoxAnchorBlockEvent {
```

Comment:
@kantai did you intend to include this in the event payload? It looks to be unused right now

Reply:
Ah -- no, at one point I was going to, but it would have required some cloning of sizable structs (the reward set). Removed in 99730e0c6.

Reply:
That data looked really helpful though! Right now the stacker list data in the event is just this this right?
```rs
pub struct RewardSet {
    pub rewarded_addresses: Vec<PoxAddress>,
    pub start_cycle_state: PoxStartCycleInfo,
}
```

Reply:
It'll also contain:

```rust
   // only generated for nakamoto reward sets
   pub signers: Option<Vec<NakamotoSignerEntry>>,
```

Which will contain signing keys, stacked amount for each signing key, and the number of signing slots.